policies/github/member.rego

```diff
@@ -2,7 +2,7 @@ package member
 
```

Comment:
commenting here to make it visible:
please go through actions.rego as well. 
line 46